docs/website/BuildTheDocumentationLocally.md

```diff
@@ -16,9 +18,20 @@ sudo apt install libasound2 xvfb
 ```
 
 ## Preview the documentation locally
-- Execute `docs/BuildLocally/InstalldepsForDocBuild.sh`
-- Add `-DBUILD_DOCS=TRUE` to your cmake options
-- Build target `localSite`
-- Open the link [localhost](http://127.0.0.1:8000/)
-- Now you should see a live preview of the docs
+- Open the Windows PowerShell (click the windows button on your keyboard and type

```

Comment:
Do we really need this multiline version? 
At my pc it is sufficient to double klick at `docs/BuildLocally/InstalldepsForDocBuild.sh`